prbench/src/prbench/envs/dynamic3d/objects.py

```diff
@@ -345,6 +349,14 @@ def __repr__(self) -> str:
             f"size={self.size}, rgba='{self.rgba}', mass={self.mass})"
         )
 
+    def get_object_centric_data(self) -> dict[str, float]:
+        data = super().get_object_centric_data()
+        # Add extents.
+        data["x_extent"] = 2 * self.size[0]
+        data["y_extent"] = 2 * self.size[1]
+        data["z_extent"] = 2 * self.size[2]
```

Comment:
Can we change this to bounding box, instead of extent? Extent is around a centroid, which may or may not be the x, y, z pos in the state (assuming we are sticking with the pos of the object's local frame in the mujoco sim as the state) - mujoco's primitive shapes are an exception. This means that the extent will not give us full information about the object, but a bounding box will.